data/human/Republic Paradise Threatened.TXT

```diff
@@ -0,0 +1,301 @@
+			`	Agent Norris steps calmly forward and presses the button to respond.
+			`	"<destination> control," he says, his voice smooth as silk, completely unlike his usual gravelly rumble. "This is <ship>, currently acting on parliamentary orders. I am transmitting a warrant which allows us to act as we see fit. I understand that you will want to check it with your superiors, so be my guest."`
+			`	He cuts the comms and then leans over the nav screen, his eyes narrowed as he searches for the mine.`
+			`	"There," he says, pointing to a cluster of machinery at the foot of an old lava flow. "Take us in, Captain."`
+			`	The <ship> touches down and Agent Norris already has a mask on and the sampling tool in his hand. He jumps out of the hatch before the ramp has even finished extending and then pushes the tool into the ground, where it immediately separates and then seals around a sample. He strides back up the ramp and hits the button to seal the hatch.`
+			`	"That's it, Captain," he says as soon as he has taken off the breath mask. "Break atmosphere and we can move on to the next location."`
+
+
+mission "Investigate the Source B"
+	autosave
+	landing
+	name "Investigate the Source"
+	description "Track down the source of the plutonium used for the Free Worlds nukes."
+	deadline
+	passengers 1
+	priority
+	source "Lodestone"
+	destination "Placer"
+	to offer
+		has "Investigate the Source A: done"
+	on enter
+		conversation
+			`It is a relief to be back in space where you are significantly less likely to be hit by a lava bomb. Agent Norris calls up the star chart and selects the next destination.`
+			`	"<destination> next, Captain," he says. "We should expect a little more pushback this time as the warrant won't be a complete surprise. Don't worry though, they can't get an injunction this quickly."`
+			`	"So how quickly can they get one?" you ask. "And how many more systems are we looking at?"`
+			`	He looks at you for several long moments, his face expressionless.
+			`	"I won't lie to you, it will be tight."
+			choice
+				`	"Fair enough, this was never going to be easy."`
+					end
+				`	"This is sounding worse by the moment."`
+				`	"And if they get an injunction? What, we just stop?"`
+			`	For the first time since you met him Agent Norris actually smiles, a grim expression which makes you think that he really hopes the worst will happen.`
+			`	"Captain <last>," he says, leaning casually against a console. "We will complete this mission. The whole mission, not just the easy parts. The Syndicate are not above the law."`
+			`	He nods once and retires to his bunk leaving you to plot the course to <system>, his words still echoing in your head. You think you've seen a fair bit of the galaxy, much more than most of humanity, and in your experience in Syndicate space the Syndicate makes the laws. This will certainly be interesting.`

```

Comment:
```suggestion
			`	He nods once and retires to his bunk leaving you to plot the course to <system>, his words still echoing in your head. You think you've seen a fair bit of the galaxy, much more than most of humanity, and in your experience in Syndicate space the Syndicate makes the laws. This will certainly be interesting.`
			label end
```